website_sale_product_detail_attribute_image/README.rst

```diff
@@ -0,0 +1,91 @@
+===========================================
+Website Sale Product Detail Attribute Image
+===========================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge2| image:: https://img.shields.io/badge/github-OCA%2Fe--commerce-lightgray.png?logo=github
+    :target: https://github.com/OCA/e-commerce/tree/13.0/website_sale_product_detail_attribute_image
+    :alt: OCA/e-commerce
+.. |badge3| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/e-commerce-13-0/e-commerce-13-0-website_sale_product_detail_attribute_image
+    :alt: Translate me on Weblate
+.. |badge4| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/113/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| 
+
+This module extends the functionality of website sale module to allow to
+display images, name and values related to product attributes in e-commerce
+product page.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+* Go to 'Website > Settings > Configuration > Products > Attributes'.
```

Comment:
It think it's working as expected. Functional review 👍 

Just a minor change: Go to 'Website > Configuration > Products > Attributes'.